hazelcast/predicate.py

```diff
@@ -360,9 +362,10 @@ def __init__(self, predicate, page_size, comparator=None):
         self._internal_predicate = predicate
         self._page_size = page_size
         self._page = 0  # initialized to be on first page
-        self.comparator = comparator
-        self.iteration_type = IterationType.ENTRY
-        self.anchor_list = []  # List of pairs: (nearest page, (anchor key, anchor value))
+        self.comparator: typing.Any = comparator
+        self.iteration_type: int = IterationType.ENTRY
+        # List of pairs: (nearest page, (anchor key, anchor value))
+        self.anchor_list: typing.List[typing.Tuple[int, typing.Tuple[typing.Any, typing.Any]]] = []
```

Comment:
`typing.Tuple[typing.Any, typing.Any]` is equivalent to `tuple` or `typing.Tuple`, you may consider using the shorter alternatives.

Reply:
I did this on purpose, to make it more clear that it contains two elements of type any. If it is not a problem for you, I would like to keep it like that.

I also used typing.Any or None return type in places to make sure that I did not miss anything.

Reply:
BTW, we can't make use of `tuple` or `list` as they are not available in Python 3.6